test/test_condensed_downloads.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""
+Condensed Downloads Test
+"""
+import importlib
+import pytest
+import subprocess
+from test.test_data import __all__ as all_test_name
+
+# Test to check condensed files are commited according to the package test data of checkers
+def test_condensed_downloads():
+    test_data = list(
+        map(lambda x: importlib.import_module(f"test.test_data.{x}"), all_test_name)
+    )
+
+    package_names = []
+    package_test_datas = map(lambda x: x.package_test_data, test_data)
+    for package_test_data in package_test_datas:
+        for package_data in package_test_data:
+            package_names.append(
+                "test/condensed-downloads/" + package_data["package_name"] + ".tar.gz"
+            )
+
+    condensed_downloads = subprocess.run(
+        "git ls-files test/condensed-downloads",
+        shell=True,
```

Comment:
`shell=True` is typically flagged as a security issue (e.g. by Bandit, which we do use for checking before release). Please replace this with a call that worked with `shell=False`

Reply:
Got to know about why it's unsafe to use. Thanks for pointing it out!